corehq/apps/data_analytics/migrations/0002_maltrow_threshold.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('data_analytics', '0001_squashed_0004_auto_20150810_1710'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='maltrow',
+            name='threshold',
+            field=models.PositiveSmallIntegerField(default=15),
+            preserve_default=True,
```

Comment:
I'm not sure how big the `maltrow` table is but this will result in a table re-write to add the default value to all the existing rows (before dropping the default constraint again).
```sql
BEGIN;
ALTER TABLE "data_analytics_maltrow" ADD COLUMN "threshold" smallint DEFAULT 15 NOT NULL CHECK ("threshold" >= 0);
ALTER TABLE "data_analytics_maltrow" ALTER COLUMN "threshold" DROP DEFAULT;
COMMIT;
```